opm/autodiff/NonlinearSolver_impl.hpp

```diff
@@ -150,10 +150,10 @@ namespace Opm
 
         linearIterations_ += linIters;
         nonlinearIterations_ += iteration - 1; // Since the last one will always be trivial.
-        wellIterations_ += wellIters; 
+        wellIterations_ = wellIters;
```

Comment:
I think it should be ``=``, because every sub time step, well equation will be solved or not, if not, this ``+=`` will give the same number of last sub time step, I do not think that makes sense.